mypy.ini

```diff
@@ -48,81 +48,187 @@ warn_return_any = True
 #
 # disable_recursive_aliases = True
 
-[mypy-manim._config.*]
+[mypy-manim.animation.animation]
 ignore_errors = True
-disable_error_code = return-value
 
-[mypy-manim._config.cli_colors]
-ignore_errors = False
+[mypy-manim.animation.composition]
+ignore_errors = True
 
-[mypy-manim._config.logger_utils]
-ignore_errors = False
+[mypy-manim.animation.creation]
+ignore_errors = True
 
-[mypy-manim.animation.*]
+[mypy-manim.animation.growing]
 ignore_errors = True
 
-[mypy-manim.animation.changing]
-ignore_errors = False
+[mypy-manim.animation.indication]
+ignore_errors = True
 
-[mypy-manim.animation.fading]
-ignore_errors = False
+[mypy-manim.animation.movement]
+ignore_errors = True
 
-[mypy-manim.animation.numbers]
-ignore_errors = False
+[mypy-manim.animation.rotation]
+ignore_errors = True
 
-[mypy-manim.animation.updaters.update]
-ignore_errors = False
+[mypy-manim.animation.speedmodifier]
+ignore_errors = True
 
-[mypy-manim.camera.*]
+[mypy-manim.animation.transform_matching_parts]
 ignore_errors = True
 
-[mypy-manim.cli.*]
-ignore_errors = False
+[mypy-manim.animation.transform]
+ignore_errors = True
 
-[mypy-manim.cli.cfg.*]
-ignore_errors = False
+[mypy-manim.animation.updaters.mobject_update_utils]
+ignore_errors = True
 
-[mypy-manim.gui.*]
+[mypy-manim.camera.camera]
 ignore_errors = True
 
-[mypy-manim.mobject.*]
+[mypy-manim.camera.mapping_camera]
 ignore_errors = True
 
-[mypy-manim.mobject.frame]
-ignore_errors = False
+[mypy-manim.camera.moving_camera]
+ignore_errors = True
 
-[mypy-manim.mobject.geometry.*]
+[mypy-manim.camera.multi_camera]
 ignore_errors = True
 
-[mypy-manim.mobject.graphing.scale]
-ignore_errors = False
+[mypy-manim.camera.three_d_camera]
+ignore_errors = True
 
-[mypy-manim.mobject.text.code_mobject]
-ignore_errors = False
+[mypy-manim._config.utils]
+ignore_errors = True
```

Comment:
Could this be moved back to the top? When sorting files and directories, usually those starting with an underscore go first.